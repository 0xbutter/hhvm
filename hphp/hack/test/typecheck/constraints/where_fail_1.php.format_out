Internal error in /data/users/drphil/fbsource/fbcode/hphp/hack/test/typecheck/constraints/where_fail_1.php
