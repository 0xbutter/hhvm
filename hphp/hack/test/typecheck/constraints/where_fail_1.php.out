File "where_fail_1.php", line 14, characters 28-29:
Some type constraint(s) here are violated (Typing[4110])
File "where_fail_1.php", line 10, characters 56-56:
'T' is a constrained type
File "where_fail_1.php", line 10, characters 61-67:
This is an object of type Derived
File "where_fail_1.php", line 13, characters 24-27:
It is incompatible with an object of type Base
